sc-poc/managers/configman/adapter/adapter.go

```diff
@@ -17,7 +17,7 @@ type Adapter interface {
 	GetRawConfig() (common.ConfigType, error)
 
 	// List returns all registered sources of a specific source type
-	List(schema.GroupVersionResource) (*unstructured.UnstructuredList, error)
+	List(schema.GroupVersionResource, string) (*unstructured.UnstructuredList, error)
```

Comment:
This should be a struct called `ListOptions` which one filter named `Labels` which could be a `map[string]string`